src/h5web/visualizations/containers/HeatmapVisContainer.tsx

```diff
@@ -20,10 +20,6 @@ function HeatmapVisContainer(props: VisContainerProps): ReactElement {
   }
 
   const value = useDatasetValue(entity.id);
-  if (!value) {
-    return <></>;
-  }
```

Comment:
`useDatasetValue` throws if `value` is being fetched. If for some reason we receive an `undefined` value, then the assertion in the mapped vis component or in the vis component (for `ScalarVis`) will throw.